umbra/umbra_64.c

```diff
@@ -354,6 +363,18 @@ static ptr_uint_t map_disp_win81[] = {
  * We check conflicts later when creating shadow memory.
  */
 #ifdef LINUX
+#ifdef AARCH64
+static const app_segment_t app_segments_initial[] = {
+    {(app_pc)0x0000ff0000000000,  (app_pc)0x0001000000000000, 0},
```

Comment:
Just having another look, and I don't understand why the layout needs to be different from the existing one. Can you explain, or provide a reference, regarding how you defined this app segment layout please?

Reply:
Okay I will review this section and get back to you.

Reply:
Okay so I when you start up DRMemory does a scan of all the current memory blocks. I am pretty sure  I simply looked at the default ones I was getting every time on my taishan AArch64 system and copied them in.